server/api/boundaries/summary.get.ts

```diff
@@ -188,6 +197,10 @@ export default defineEventHandler(async (event) => {
       });
     }
 
+    if (!uniqueFinding.Stigs[0].id) {
```

Comment:
After discussion, seems like this hack is going to be removed once the functionality is changed to create the assessment information upon the addition of a stig to a system.